lexers/m/mysql.go

```diff
@@ -51,4 +55,10 @@ var MySQL = internal.Register(MustNewLexer(
 			{`"`, LiteralStringDouble, Pop(1)},
 		},
 	},
-))
+).SetAnalyser(func(text string) float32 {
+	if mysqlAnalyzerNameBetweenBacktickRe.MatchString(text) {
+		return 0.1
```

Comment:
This is what I mean.
```suggestion
		return 0.2
```